keybase/platform_darwin.go

```diff
@@ -183,6 +183,12 @@ func (c context) stop() error {
 		c.log.Warningf("Error requesting app exit: %s", appExitErr)
 	}
 
+	// Stop the redirector so it can be upgraded for all users.
+	_, redirectorErr := command.Exec(c.config.keybasePath(), []string{"uninstall", "--components=redirector"}, time.Minute, c.log)
```

Comment:
Is `c.config.keybasePath()` the `keybase` binary? Out of curiosity I did `keybase uninstall` and it seemed to be removing a bunch of binaries from my system as part of uninstalling the app. So it sounds like `keybase uninstall` is different from "uninstalling" a mount?

Reply:
> Is c.config.keybasePath() the keybase binary?

Yep.

> So it sounds like keybase uninstall is different from "uninstalling" a mount?

`keybase uninstall` without any flags attempts to do a full uninstall.  But if you give it `--components`, you can do smaller things, like stopping KBFS, or uninstalling just the fuse module, or deleting just the mount directory.  So I added a component for stopping the redirector.

Reply:
Got you. Thanks for explaining it. So there are two levels of "uninstall":

1. Stop the redirector and removing the dir that corresponds to the mount point.

2. Remove the keybase-redirector itself.

It seems uninstalling the `redirector` component is 1. I suppose we still need 2 somewhere?

Reply:
> It seems uninstalling the redirector component is 1. I suppose we still need 2 somewhere?

Maybe, though it would be as part of one of the other PRs in client.  I can test it out and see if that binary is treated differently from the others.

Reply:
Got it; thanks!